resources/api/reservation.py

```diff
@@ -260,6 +263,14 @@ def to_representation(self, instance):
     def get_is_own(self, obj):
         return obj.user == self.context['request'].user
 
+    def get_user_permissions(self, obj):
```

Comment:
Wonder if it'd be worth to just `raise SkipField` if this'd return false, so as to save a chunk of bandwidth...